proposals/relaxed-simd/Overview.md

```diff
@@ -205,6 +205,27 @@ def blend(a : v128, b : v128, m: v128, lanes : int):
   return result
 ```
 
+### Relaxed min and max
+
+- `f32x4.min(a: v128, b: v128) -> v128`
+- `f32x4.max(a: v128, b: v128) -> v128`
+- `f64x2.min(a: v128, b: v128) -> v128`
+- `f64x2.max(a: v128, b: v128) -> v128`
+
+Return the lane-wise minimum or maximum of two values. If either values is NaN, or the values are -0.0 and +0.0, the return value is implementation-defined.
+
+```python
+def min_or_max(a : v128, b : v128, lanes : int, is_min : bool):
+  result = []
+  for i in range(lanes):
+    if isNaN(a[i]) or isNaN(b[i]):
+      result[i] = UNDEFINED
+    elif (a[i] == -0.0 && b[i] == +0.0) or (a[i] == +0.0 && b[i] == -0.0):
+      result[i] = UNDEFINED
```

Comment:
We need to be more constrained here, i.e. document that the return value is an implementation-defined choice function of `a[i]`, `b[i]`

Reply:
Good point, updated, ptal.